Calculator/Token_stream.cpp

```diff
@@ -30,6 +29,14 @@ Token Token_stream::get()
     }
     char ch;
     cin >> ch;
+    if(ch != print)
```

Comment:
might can evaluate these in one line...if it evaluates print and fails, will it still evaluate the get. if it doesn't evaluate get and then does unget, we have a problem

Reply:
maybe dont worry about it unless you really want to see what happens